### ISSUE TITLE
Trouble installing dependencies

### ISSUE THREAD

**Issue by mnemanja** [jakemmarsh/angularjs-gulp-browserify-boilerplate#176]
After updating the package.json to 1.6.5 and doing a clean npm install in my project 

I get this message at the end:

```
> angularjs-gulp-browserify-boilerplate@1.5.6 webdriver-update C:\www\project\client.js\src\main\webapp
> ./node_modules/gulp-protractor/node_modules/protractor/bin/webdriver-manager update

'.' is not recognized as an internal or external command,
operable program or batch file.
```
Basically running the `npm run webdriver-update` as the last command achieves the same thing.

I'm attaching the **npm-log** from the `npm install` 
[npm-debug.docx](https://github.com/jakemmarsh/angularjs-gulp-browserify-boilerplate/files/163217/npm-debug.docx)

After running `gulp protractor` I get this error:
```
[14:25:42] Starting 'protractor'...
[launcher] Process exited with error code 1
c:\www\project\client.js\src\main\webapp\node_modules\gulp-protractor\node_modules\protractor\node_modules\q\q.js:126
                    throw e;
                    ^

Error: No selenium server jar found at the specified location (c:\www\project\client.js\src\main\webapp\node_modules\gulp-protractor\node_modules\protractor\selenium\selenium-server-standalone-2.48.2.jar). Check that the version number is up to date.
    at [object Object].LocalDriverProvider.addDefaultBinaryLocs_ (c:\www\project\client.js\src\main\webapp\node_modules\gulp-protractor\node_modules\protractor\lib\driverProviders\local.js:37:11)
    at [object Object].LocalDriverProvider.setupEnv (c:\www\project\client.js\src\main\webapp\node_modules\gulp-protractor\node_modules\protractor\lib\driverProviders\local.js:71:8)
    at [object Object].Runner.run (c:\www\project\client.js\src\main\webapp\node_modules\gulp-protractor\node_modules\protractor\lib\runner.js:274:31)
    at [object Object].TaskRunner.run (c:\www\project\client.js\src\main\webapp\node_modules\gulp-protractor\node_modules\protractor\lib\taskRunner.js:123:19)
    at createNextTaskRunner (c:\www\project\client.js\src\main\webapp\node_modules\gulp-protractor\node_modules\protractor\lib\launcher.js:216:20)
    at c:\www\project\client.js\src\main\webapp\node_modules\gulp-protractor\node_modules\protractor\lib\launcher.js:239:7
    at _fulfilled (c:\www\project\client.js\src\main\webapp\node_modules\gulp-protractor\node_modules\protractor\node_modules\q\q.js:797:54)
    at self.promiseDispatch.done (c:\www\project\client.js\src\main\webapp\node_modules\gulp-protractor\node_modules\protractor\node_modules\q\q.js:826:30)
    at Promise.promise.promiseDispatch (c:\www\project\client.js\src\main\webapp\node_modules\gulp-protractor\node_modules\protractor\node_modules\q\q.js:759:13)
    at c:\www\project\client.js\src\main\webapp\node_modules\gulp-protractor\node_modules\protractor\node_modules\q\q.js:573:44

c:\www\project\client.js\src\main\webapp\gulp\tasks\protractor.js:33
            throw err;
            ^
Error: protractor exited with code 1
```

Below you can also find my package.json example. Thanks for the help a lot!

```
"engines": {
        "node": "~4.2.x"
    },
    "devDependencies": {
        "angular": "^1.5.0",
        "angular-mocks": "^1.3.15",
        "angular-ui-router": "^0.2.13",
        "babel-core": "^6.3.26",
        "babel-eslint": "^5.0.0",
        "babel-preset-es2015": "^6.3.13",
        "babel-register": "^6.5.2",
        "babelify": "^7.2.0",
        "brfs": "^1.2.0",
        "browser-sync": "^2.7.6",
        "browserify": "^13.0.0",
        "browserify-istanbul": "^1.0.0",
        "browserify-ngannotate": "^2.0.0",
        "bulk-require": "^0.2.1",
        "bulkify": "^1.1.1",
        "debowerify": "^1.3.1",
        "del": "^2.1.0",
        "eslint": "2.2.0",
        "express": "^4.13.3",
        "gulp": "^3.9.0",
        "gulp-angular-templatecache": "^1.3.0",
        "gulp-autoprefixer": "^3.1.0",
        "gulp-changed": "^1.0.0",
        "gulp-eslint": "^2.0.0",
        "gulp-gzip": "^1.2.0",
        "gulp-if": "^2.0.0",
        "gulp-imagemin": "^2.4.0",
        "gulp-notify": "^2.0.0",
        "gulp-protractor": "^2.1.0",
        "gulp-rename": "^1.2.0",
        "gulp-sass": "^2.0.4",
        "gulp-sourcemaps": "^1.6.0",
        "gulp-streamify": "^1.0.2",
        "gulp-uglify": "^1.4.2",
        "gulp-util": "^3.0.1",
        "imagemin-pngcrush": "^4.1.0",
        "isparta": "^4.0.0",
        "karma": "^0.13.21",
        "karma-browserify": "^5.0.2",
        "karma-chrome-launcher": "^0.2.1",
        "karma-coverage": "douglasduteil/karma-coverage#next",
        "karma-firefox-launcher": "^0.1.3",
        "karma-jasmine": "^0.3.6",
        "karma-sauce-launcher": "^0.3.0",
        "merge-stream": "^1.0.0",
        "pretty-hrtime": "^1.0.1",
        "run-sequence": "^1.1.5",
        "tiny-lr": "^0.2.1",
        "uglifyify": "^3.0.1",
        "vinyl-buffer": "^1.0.0",
        "vinyl-source-stream": "^1.1.0",
        "watchify": "^3.7.0",

        "angular-animate": "1.5.0",
        "angular-bootstrap-npm": "0.13.0",
        "angular-cookies": "1.5.0",
        "angular-file-upload-shim": "1.6.4",
        "angular-messages": "1.5.0",
        "angular-motion-sass": "0.0.1",
        "angular-strap": "2.2.0",
        "autosize": "3.0.14",
        "bootstrap": "3.3.4",
        "gulp-sftp": "0.1.5",
        "gulp-war": "0.1.2",
        "gulp-zip": "3.0.2",
        "jquery": "2.1.4",
        "jshint-stylish": "2.1.0",
        "lodash": "3.9.3",
        "ng-file-upload": "7.0.10",
        "ng-sortable": "1.3.1",
        "pix-diff": "1.0.7",
        "restangular": "1.5.1",
        "round-to": "^1.1.0",
        "sha256": "0.2.0",
        "simple-statistics": "1.0.1"
    },
    "scripts": {
        "test": "./node_modules/.bin/gulp test",
        "webdriver-update": "./node_modules/gulp-protractor/node_modules/protractor/bin/webdriver-manager update",
        "postinstall": "npm run webdriver-update"
    }
```

**Comment by mnemanja** [comment]
And yes, I'm running it on windows.

**Comment by karneaud** [comment]
+1

**Comment by jakemmarsh** [comment]
What happens if you change the `webdriver-update` script back to what it was before https://github.com/jakemmarsh/angularjs-gulp-browserify-boilerplate/commit/915c81be6c6d20d4517eec62ea1170c4ac1b874c? I recently changed the script and that could be causing issues (although it doesn't for me locally)

**Comment by mnemanja** [comment]
Then I get this:

```
'webdriver-manager' is not recognized as an internal or external command,
operable program or batch file.

npm ERR! Windows_NT 6.1.7601
npm ERR! argv "C:\\Program Files\\nodejs\\node.exe" "C:\\Users\\Nemanja.Milosavljevi\\AppData\\Roaming\\npm\\node_modules\\npm\\bin\\npm-cli.js" "run" "webdriver-update"
npm ERR! node v4.2.1
npm ERR! npm  v3.5.2
npm ERR! code ELIFECYCLE
npm ERR! angularjs-gulp-browserify-boilerplate@1.5.6 webdriver-update: `webdriver-manager update`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the angularjs-gulp-browserify-boilerplate@1.5.6 webdriver-update script 'webdriver-manager update'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the angularjs-gulp-browserify-boilerplate package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     webdriver-manager update
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs angularjs-gulp-browserify-boilerplate
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls angularjs-gulp-browserify-boilerplate
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     C:\www\project\client.js\src\main\webapp\npm-debug.log

npm ERR! Windows_NT 6.1.7601
npm ERR! argv "C:\\Program Files\\nodejs\\node.exe" "C:\\Program Files\\nodejs\\node_modules\\npm\\bin\\npm-cli.js" "install"
npm ERR! node v4.2.1
npm ERR! npm  v2.14.7
npm ERR! code ELIFECYCLE
npm ERR! angularjs-gulp-browserify-boilerplate@1.5.6 postinstall: `npm run webdriver-update`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the angularjs-gulp-browserify-boilerplate@1.5.6 postinstall script 'npm run webdriver-update'.
npm ERR! This is most likely a problem with the angularjs-gulp-browserify-boilerplate package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     npm run webdriver-update
npm ERR! You can get their info via:
npm ERR!     npm owner ls angularjs-gulp-browserify-boilerplate
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     C:\www\project\client.js\src\main\webapp\npm-debug.log
```

**Comment by mnemanja** [comment]
This is when the "web-driver" line was changed: https://github.com/jakemmarsh/angularjs-gulp-browserify-boilerplate/issues/160

**Comment by jakemmarsh** [comment]
Please pull master and try a fresh start. This is hopefully fixed in #180

**Comment by mnemanja** [comment]
Good job @jakemmarsh and thanks a lot :+1:
